ui/src/app/change_password/page.tsx

```diff
@@ -0,0 +1,81 @@
+    const handlePassword2Change = (event: ChangeEvent<HTMLInputElement>) => { setPassword2(event.target.value) }
+    return (
+        <div style={{
+            display: "flex",
+            alignContent: "center",
+            justifyContent: "center",
+            flexWrap: "wrap",
+            height: "100vh",
+        }}>
+            <div className="p-panel" style={{
+                width: "45rem",
+                minWidth: "min-content",
+                minHeight: "min-content",
+            }}>
+                <div className="p-panel__header is-sticky">
+                    <h4 className="p-panel__title">Change Password</h4>
+                </div>
+                <div className="p-panel__content">
+                    <div className="u-fixed-width">
+                        <form className={"p-form-validation " + (!passwordsMatch ? "is-error" : "")}>
+                            <label className="p-form__label">Password</label>
+                            <input className="p-form-validation__input" type="password" id="password1" name="password" placeholder="******" autoComplete="current-password" required={true} onChange={handlePassword1Change} />
+                            <p className="p-form-help-text">
+                                Password must have 8 or more characters, must include at least one capital letter, one lowercase letter, and either a number or a symbol.
+                            </p>
+                            <label htmlFor="p-form__label">Password Again</label>
+                            <input className="p-form-validation__input" type="password" id="InputPassword" name="password2" placeholder="******" autoComplete="current-password" onChange={handlePassword2Change} />
+                            {!passwordsMatch &&<p className="p-form-validation__message">Passwords do not match</p>}
+                            {errorText &&
+                                <div className="p-notification--negative">
+                                    <div className="p-notification__content">
+                                        <h5 className="p-notification__title">Error</h5>
+                                        <p className="p-notification__message">{errorText.split("error: ")}</p>
+                                    </div>
+                                </div>
+                            }
+                            {!passwordsMatch ? (
+                                <>
+                                    <button type="submit" name="submit" disabled={true}>Submit</button>
+                                </>
+                            ) : (
+                                <button type="submit" name="submit" onClick={(event) => { event.preventDefault(); mutation.mutate({ authToken: (auth.user ? auth.user.authToken : ""), username: (auth.user ? auth.user.username : ""), password: password1 }) }}>Submit</button>
```

Comment:
Can we move the submit button to the right and make it green?

I listed enhancement recommendations for the login page and some of those can be applied here as well. Including the submit button position and color. 

- https://github.com/canonical/gocert/issues/47

Reply:
I've added some of the changes from there to this form, please take a look again.

The only thing from this issue i may push back on is putting the button on the right. English text starts from the left, so it's usual for readers to look at the left of the row for the button under a form. I personally like it there too.

also added a show button to see the password for the first field.

I will take this issue's ticket after all of the queued up changes are merged.

Reply:
I'm ok with keeping it on the left.